onlyoffice/.gitignore

```diff
@@ -0,0 +1,8 @@
+coverage.out
```

Comment:
file is not needed. Have a look at the top level `.gitignore` if something needs to be added.